docs/admin/install/knative-with-operators.md

```diff
@@ -328,9 +328,14 @@ kubectl logs -f deploy/knative-operator
     kubectl apply --filename {{artifact(repo="serving",file="serving-default-domain.yaml")}}
     ```
 
-    **Caveat**: This will only work if the cluster LoadBalancer service exposes an
-    IPv4 address or hostname, so it will not work with IPv6 clusters or local setups
-    like Minikube. For these, see "Real DNS" or "Temporary DNS".
+    !!! info "Caveat"
+        This will only work if the cluster `LoadBalancer` service exposes an
```

Comment:
Is `LoadBalancer` the literal name of the service? Otherwise I'd write "load balancer". Also, service -> Service as per https://dev-knative.netlify.app/development/help/contributor/style-guide/word-and-phrase-list/ and Minikube -> minikube

Reply:
yes, it's literally a LoadBalancer service type (https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer), but again I didn't change this